recipes/rmats2sashimiplot/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: rmats2sashimiplot
+  version: "2.0.0"
+
+source:
+  git_rev: c3690679de217238b47251c574b3ec2b89af5745
+  git_url: https://github.com/Xinglab/rmats2sashimiplot.git
+
+requirements:
+  build:
+    - python ==2.6*|2.7*
+  run:
+    - python ==2.6*|2.7*
+    - matplotlib
+    - scipy
+    - pysam ==0.9.0
+    - samtools ==1.3.1
+
+build:
+  noarch: python
+  preserve_egg_dir: True
+
+about:
+  home: https://github.com/Xinglab/rmats2sashimiplot
```

Comment:
Add a `license` directive (in this case should be `license: GNU GPLv2`, as well as a `license_file`, which is available as `license_file: LICENSE`.
Also add a short summary under the `summary` directive.